src/components/ThemeHandler.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useMemo, useState } from "react";
+import CssBaseline from "@mui/material/CssBaseline";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import { ColorModeContext } from "../context/appContext";
+import { baseTheme } from "./Themes";
+const savedMode = window.localStorage.getItem("colorMode") as "light" | "dark";
+export default function ThemeHandler({ children }: any) {
```

Comment:
you can add the following interface to fix the type warning
interface IProps {
    children: React.ReactNode;
  };
export default function ThemeHandler({ children }: IProps) {